main.cpp

```diff
@@ -19,6 +20,24 @@ int main(int argc, char *argv[])
 
     QApplication app(argc, argv);
 
+    QSettings settings(QDir(QDir::home().filePath(".john")).filePath("johnny.conf"),
```

Comment:
Let's pass this settings var to MainWindow instead of initializing another one for that.